wireapp/wire-android
# feat: rename group conversations flow (AR-1112)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1112" title="AR-1112" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1112</a>  Edit Group Name 
  </td></table>
  <br />
 
























----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Implementation of user flow to edit group names.

### Solutions

Refactor existing components (group creation) to support reusage of ui elements.
After the refactor, the code for this edition screen was minimal and only had to create some logic for this specific viewmodel.

### Dependencies (Optional)

- https://github.com/wireapp/kalium/pull/1106

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Go to the screen of a group conversation detail (admin rights) and navigate to group name edition, change the name, you should see the new name and a system message indicating that you renamed the conversation to "X"

### Attachments (Optional)


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
